paper/src/main/java/vg/civcraft/mc/citadel/activity/ActivityMapTimePoll.java

```diff
@@ -0,0 +1,97 @@
+package vg.civcraft.mc.citadel.activity;
+
+import java.util.concurrent.ConcurrentLinkedQueue;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.ScheduledThreadPoolExecutor;
+import java.util.concurrent.TimeUnit;
+
+class ActivityMapTimePoll {
+	private static final long POLL_INTERVAL_MS = 1000L;
+
+	private final ConcurrentLinkedQueue<Long> nanoTimes = new ConcurrentLinkedQueue<>();
+	private final ScheduledExecutorService scheduler = new ScheduledThreadPoolExecutor(1);
+	private final Object syncRoot = new Object();
+
+	private long regionLoadCount;
+	private long regionLoadSumNano;
+	private long regionLoadMinTimeNano;
+	private long regionLoadMaxTimeNano;
+
+	private void poll() {
+		long currentRegionLoadCount = 0;
+		long currentRegionLoadSumNano = 0;
+		long currentRegionLoadMinTimeNano = Long.MAX_VALUE;
+		long currentRegionLoadMaxTimeNano = Long.MIN_VALUE;
```

Comment:
Are these the right way around? Min has Long.MAX and vice versa

Reply:
I've added these values just for the self-check.
You are right, if **min** will have Long.MAX and **max** will have Long.MIN then it means that something went wrong.

The code ignores these values if currentRegionLoadCount == 0:

this: https://github.com/CivMC/Citadel/pull/39/files#diff-d5763ed05cf9027e717e5d6821f1c486b7a3600c83423f67f0c0a47ff0b12feeR28

and this: https://github.com/CivMC/Citadel/pull/39/files#diff-d5763ed05cf9027e717e5d6821f1c486b7a3600c83423f67f0c0a47ff0b12feeR44